src/components/PageNotFoundScreen/PageNotFoundScreen.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "./PageNotFoundScreen.css";
+
+export default function PageNotFoundScreen() {
+  return (
+    <div id="pagenotfound-screen">
+      <h1> 404 </h1>
+      <h1> This is just a placeholder. We might want to have a link here to let the user try logging in again or go home. </h1>
```

Comment:
I think a simple "The requested page not found" would suffice.

Reply:
Probably. This can be discussed when we work on actually building that page.